05week/checkers.js

```diff
@@ -60,7 +108,12 @@ function Game() {
 
   this.start = function() {
     this.board.createGrid();
-    // Your code here
+    this.board.createCheckers();
+    this.moveChecker = function(start, end) {
+      let checker = this.board.selectChecker(start[0],start[1]);
+      this.board.grid[start[0]][start[1]] = null;
+      this.board.grid[end[0]][end[1]] = checker;
```

Comment:
You created a `killChecker` method in `function Board` but you don't use it in this `Game`...